webpack.config.js

```diff
@@ -1,8 +1,9 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const BomPlugin = require('webpack-utf8-bom');
 
 module.exports = {
-  mode: 'development',
+  mode: 'production',
```

Comment:
Can you still debugging the scripts if you set this variable to production? If not, maybe we can create two configurations, one for local debugging and this one for production. Anyway, it's not a crucial thing.